static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -98,7 +101,11 @@ const FeedbackListItem = forwardRef<HTMLAnchorElement, Props>(
         <div style={{gridArea: 'message'}}>
           <TextOverflow>{feedbackItem.message}</TextOverflow>
         </div>
-        <Flex style={{gridArea: 'icons'}}>
+        <Flex style={{gridArea: 'icons'}} gap={space(1)} align="center">
+          <ProjInfoContainer>
+            <PlatformIcon size={16} platform={platform ?? 'default'} />
+            <TextOverflow>{slug}</TextOverflow>
+          </ProjInfoContainer>
           {feedbackItem.replay_id ? <ReplayBadge /> : null}
```

Comment:
This `<ReplayBadge>` could be some kind of BaseAvatar or idk. there's a bunch of things in there, not all are easy to use.